packages/ui/src/components/Button/index.tsx

```diff
@@ -9,7 +9,9 @@ export interface ButtonProps {
   className?: string;
   onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
   children: React.ReactNode;
-  size?: 'small' | 'medium';
+  size?: 'small' | 'medium' | 'large' | '';
```

Comment:
Mind taking a second and explain what makes us need the `''`? Cant we default to one of the sizes, perhaps medium?

Reply:
Until now I have seen Vercel has 4 types of buttons 'small' | 'medium' | 'large' and another one, the one they use for the `Login` and `SignUp` buttons. That is why I left it blank, so it would be our default button.

 If we don't leave a space we would have to create a new size then, I can call it `regular` and then extract the attributes than now are part of the `button` tag

Reply:
> That is why I left it blank, so it would be our default button.

Default for me means that if I wanted to use the button component I would not pass a prop regarding the size. 
So would be something like this 

```
<Button fullWidth type="primary"> this is a default button <Button/>
```

How do you think of using default?

Reply:
Exactly, that is our default button:

```
<Button fullWidth type="primary"> this is a default button <Button/>
```

So it doesn't need any size property, the attributes that modify the button are directly in the button tag on our button.styles.ts:
```
button {
    height: 40px;
    ...
    ...
    }
```

Reply:
right that makes sense thanks for explaining. Cant we have as default `medium`?

Reply:
Yes, I can refactor what `medium` is now so it looks like our default. Then we need another name for what `medium` is now. It is very similar but the padding is different, I will think and try to find something that makes sense

Reply:
How do you find to have `medium` for our default buttons, and `medium-sm` for the buttons that are `medium` now. Reasons?
1. They are bigger than `small` but smaller than `medium`.
2. sm is widely used for buttons sizes (Chakra uses it).
3. The other names I found are not better, `regular`, `medium-small`.

Reply:
I am confused now to be honest. I think I miss something. I am checking the stories

Reply:
Okay I think this is my bad here. I didnt understand what you said when you meant there is another button. I thought you were talking about these ones... 
![image](https://user-images.githubusercontent.com/4181674/110176395-1d7e1e80-7dfb-11eb-88ec-898c31328b73.png)
Sorry dude. 

okay well now I see what you mean... 

Okay sorry that I missed that bit again. Is a good opportunity to rename them anyway. 

What about `sm` `md` `lg` `xl`?

Reply:
I thought something was weird in what we were saying ;). It is all good, of course. We can totally do that and use `sm` `md` `lg` `xl`. The only weird thing is that in that sequence or default button would be `lg` if we consider their height and width. If you are ok with `lg` being our default I don`t mind.

Reply:
If we make `md` our default it would be weird because `md` is going to be bigger than `lg` and that doesn't really make sense I suppose.

Reply:
right thanks for explaining ma man. okay lets fo then this `xs` `sm` `md` `lg`. This then would make sense for both of us, or??

Reply:
Yes!! That is the best solution, I am the one to have to be sorry for not presenting that solution in my answer, really.

I think I have to be so concentrated new in the things I am learning that I am missing the obvious. But not excuses! More work is what I have to do. Thanks for the patience man!